src/main/java/frc/robot/subsystems/SwerveDrivebase.java

```diff
@@ -340,25 +342,24 @@ public void setModulesNeutralMode(NeutralModeValue neutralModeValue) {
         }
     }
 
-
-    public Command badJankAlignWithPoint () {
+    public Command badJankAlignWithPoint() {
         return Commands.runEnd(() -> {
             double maxLinearSpeed = 1.5;
             Rotation2d maxRotationalSpeed = Rotation2d.fromDegrees(120);
             Pose2d error = nearest.relativeTo(getPose2d());
             // This math sometimes overrruns and does 360 noscopes
-            Rotation2d rotError = error.getRotation().times(5);
-            if(Math.abs(rotError.getRadians()) > maxRotationalSpeed.getRadians()) {
-                rotError = rotError.times(maxRotationalSpeed.getRadians()/rotError.getRadians());
+            Rotation2d rotError = error.getRotation().times(2.5);
+            if (Math.abs(rotError.getRadians()) > maxRotationalSpeed.getRadians()) {
+                rotError = rotError.times(maxRotationalSpeed.getRadians() / rotError.getRadians());
             }
-            Translation2d posError = error.getTranslation().times(4);
-            if(posError.getNorm() > maxLinearSpeed) {
+            Translation2d posError = error.getTranslation().times(2);
+            if (posError.getNorm() > maxLinearSpeed) {
                 // limit max speed
```

Comment:
We will maybe have to change these back to be more responsive tomorrow with testing

Reply:
*almost certainly